src/stores/game.js

```diff
@@ -230,7 +230,6 @@ export const useGameStore = defineStore('game', () => {
       typeof lastEventPlayer === 'number' && myPNum.value !== null
         ? lastEventPlayer === myPNum.value
         : null;
-    lastEventDiscardedCards.value = newGame.lastEvent?.discardedCards ?? null;
```

Comment:
Removed lastEventDiscardedCards from state as we now pull them directly from the socket event